Resources/Prototypes/Recipes/Construction/utilities.yml

```diff
@@ -46,6 +46,21 @@
   placementMode: SnapgridCenter
   canBuildInImpassable: false
 
+- type: construction
+  name: AME shielding
+  id: AMEShielding
+  graph: ameShielding
+  startNode: start
+  targetNode: ameShielding
+  category: Utilities
+  description: "Shielding for the antimatter engine"
+  icon:
+    sprite: Structures/Power/Generation/ame.rsi
+    state: shield_0
+  objectType: Structure
+  placementMode: SnapgridCenter
+  canBuildInImpassable: false
```

Comment:
Honestly, just remove this. The AME shielding should not be in the construction menu.